sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -162,10 +162,9 @@ void _remove_tentative_addr(gnrc_netif_t *netif, const ipv6_addr_t *addr)
         /* Cannot use target address as personal address and can
          * not change hardware address to retry SLAAC => use purely
          * DHCPv6 instead */
-        /* TODO: implement IA_NA for DHCPv6 */
-        /* then => tgt_netif->aac_mode = GNRC_NETIF_AAC_DHCP; */
-        DEBUG("nib: would set interface %i to DHCPv6, "
-              "but is not implemented yet", netif->pid);
+
+        /* TODO: How will this flag be used? */
+        netif->ipv6.aac_mode = GNRC_NETIF_AAC_DHCP;
```

Comment:
This should be kept as is. The point of this line is that SLAAC (AAC_AUTO) failed and we now need to fallback to DHCP.  However, we still need a fallback in case there is no DHCP (which we can check using `IS_USED(MODULE_DHCPV6_CLIENT_IA_NA)`)

Reply:
(the fallback should be the old code, if that was not clear).

Reply:
Hmm, so should this look something like this?

```
        IS_USED(MODULE_DHCPV6_CLIENT_IA_NA) {
            netif->ipv6.aac_mode |= GNRC_NETIF_AAC_DHCP;
        } else {
            /* Perform alternative fallback */
        }
```